cmake/install-dlls-mingw.cmake.in

```diff
@@ -23,6 +24,7 @@ set(MINGW_CORE_DLLS
     "version.dll"
     "winmm.dll"
     "ws2_32.dll"
+	
```

Comment:
Can you fix these two? Misalignment and a needless empty line.

Reply:
Yeah somehow it really depends on the editor. Notepad++ did not show me any difference.

Reply:
It depends on the tab width you have set.